images/base/Dockerfile

```diff
@@ -165,26 +165,16 @@ RUN git clone --filter=tree:0 "${CRI_TOOLS_CLONE_URL}" /cri-tools \
     && make BUILD_BIN_PATH=./build crictl \
     && GOARCH=$TARGETARCH go-licenses save --save_path=/_LICENSES ./cmd/crictl
 
-# stage for building cni-plugins
+# stage for building cni-plugin
```

Comment:
hmm, this has a risk of skewing with kindnetd right?

we probably need to think about that, though I'm not sure what the best option is.
we could modify the node image build to handle this but I'm not sure that's better